panther_analysis_tool/constants.py

```diff
@@ -16,7 +16,7 @@
 
 PACKAGE_NAME: Final = "panther_analysis_tool"
 
-VERSION_STRING: Final = "0.20.1"
+VERSION_STRING: Final = "0.20.2"
```

Comment:
since this is a new behavior. This should probably go under 0.21.0

Reply:
I considered that and wound up (incorrectly) leaning towards "it's tightly coupled to an existing feature"

Reply:
got it, yeah we haven't been super strict about this, but since we're adding a new command argument as well is what pushed me towards version bump. Ideally minor bumps should be just for bug fixes.

Reply:
yup patch, lol bad choice of words there 🤦🏾‍♂️